recipe/meta.yaml

```
@@ -41,6 +41,8 @@ requirements:
     - {{ compiler('cuda') }}  # [linux64 or ppc64le or win]
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - sysroot_linux-64 2.17  # [linux64]
+    - libcufft-dev  # [(linux64 or ppc64le or win) and cuda_compiler_version.startswith("12")]
+    - cuda-profiler-api  # [(linux64 or ppc64le or win) and cuda_compiler_version.startswith("12")]
```

Sorry maybe using a Jinja filter is better?

```suggestion
    - libcufft-dev  # [(linux64 or ppc64le or win) and cuda_compiler_version|startswith("12")]
    - cuda-profiler-api  # [(linux64 or ppc64le or win) and cuda_compiler_version|startswith("12")]
```

Alternatively could try

```suggestion
    - libcufft-dev  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
    - cuda-profiler-api  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
```